src/Frontend/Themes/Fork/Core/Layout/Templates/Base.html.twig

```diff
@@ -6,7 +6,7 @@
     {% include "Core/Layout/Templates/Cookies.html.twig" %}
   {% endblock %}
 
-  <div class="header header-blue" style="background-image: url('{{ FRONTEND_FILES_URL }}/Pages/images/source/{{ page.data.image }}');">
+  <div class="header header-blue" {% if page.data.image is defined and page.data.image is not empty %}style="background-image: url('{{ FRONTEND_FILES_URL }}/Pages/images/source/{{ page.data.image }}');"{% endif %}>
```

Comment:
I think this can easier: {% if page.data.image %} ?

Reply:
that will result in an error if page.data.image doesn't exist if I remember, or is that only when page for instance wouldn't exist?

Reply:
i think, but i am not sure